### PR TITLE
[improve][doc][Connector-V2][http] improve http doc param body des 

### LINKED ISSUE #5210: [Bug] [Source Http]  params is invalid
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### What happened

Http Source  params is invalid!

### SeaTunnel Version

dev分支（2.3.2-SNAPSHOT）

### SeaTunnel Config

```conf
env {
    execution.parallelism = 5
    job.mode = "BATCH"
    job.name = "http2mysql"
}
 
source {
    Http {
        url = "http://127.0.0.1:8080/api/demo2"
        method = "POST"
        format = "json"
        params =  {
            id = 1
            }
        json_field = {
            id = "$.value[*].id"
            lrrq = "$.value[*].lrrq"
        }
        schema = {
            fields {
                id = int
                lrrq = string
            }
        }
    }

}
transform {
    
}
sink {
    console {
    }
}
```

### Running Command

```shell
./bin/seatunnel.sh -c config/http2mysql.conf -e local
```

### Error Exception

```log
no error
```

### Zeta or Flink or Spark Version

2.3.2-SNAPSHOT

### Java or Scala Version

_No response_

### Screenshots

image
image

The interface returns the following：
image

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by jackyyyyyssss**
## [improve][doc][Connector-V2][http] improve http doc param body des  
#5210 Many users may mistakenly recognize param as an HTTP JSON body supplementary parameter description

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: docs/en/connector-v2/source/Http.md [hunk 1]
```diff
@@ -42,24 +42,24 @@ They can be downloaded via install-plugin.sh or from the Maven central repositor
...
+| content_json                | String  | No       | -       | This parameter can get some json data.If you only need the data in the 'book' section, configure `content_field = "$.store.book.*"`.                                      |
+| format                      | String  | No       | json    | The format of upstream data, now only support `json` `text`, default `json`.                                                                                              |
+| method                      | String  | No       | get     | Http request method, only supports GET, POST method.                                                                                                                      |
+| headers                     | Map     | No       | -       | Http headers.                                                                                                                                                             |
+| params                      | Map     | No       | -       | Http params,the program will automatically add http header application/x-www-form-urlencoded,invoke by request.setEntity(new UrlEncodedFormEntity(parameters, ENCODING)). |
```
Comment (Hisoka-X) [inline docs/en/connector-v2/source/Http.md]:
```suggestion
| params                      | Map     | No       | -       | Http params,the program will automatically add http header application/x-www-form-urlencoded. |
```

**Comment by Hisoka-X** [comment]
If both params and body configured, what's header look like?

**Comment by jackyyyyyssss** [comment]
> If both params and body configured, what's header look like?

image
At first, both headers have an addbody method that will overwrite the headerimage

**Review by Hisoka-X — APPROVED** [review]
Thanks @jackyyyyyssss

**Comment by jackyyyyyssss** [comment]
> Thanks @jackyyyyyssss

I tested all the situations using Charles to capture packets

1. Params

2. Body

3. Both  have the same HTTP header
135ab999090dee949e3f5fad98ca5fe




There seems to be a bug in setting params that may cause the set value to be lost. I will restart a PR to fix this issue

**Comment by Hisoka-X** [comment]
> I will restart a PR to fix this issue

Great! Thanks @jackyyyyyssss

**Review by liugddx — APPROVED** [review]

**Pull request merged**